src/components/calculator/Calculator.css

```diff
@@ -0,0 +1,71 @@
+.calculator {
+  padding: 2em;
+  margin: 10em auto;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+  gap: 1em 1em;
```

Comment:
- [ ] I think you gave unnecessarily wide gap. Kindly make it close to the original design. In addition, it would be better if you have a default 0 in your input/display field. Kindly make these changes. :pray: 
![image](https://user-images.githubusercontent.com/64894927/134194362-c9313d02-4967-4f33-b0da-d9f9edfeeb3c.png)
![image](https://user-images.githubusercontent.com/64894927/134194403-e91f50a5-feea-4aa3-a866-211de77c194a.png)